src/app/containers/ScheduleSummary.tsx

```diff
@@ -89,6 +89,8 @@ class ConnectedScheduleSummary extends React.Component<ConnectedScheduleSummaryP
             .filter(l => doTimeRangesOverlap(
                 { startTime: moment(l.startDate).toISOString(), endTime: moment(l.endDate).toISOString() },
                 { startTime: moment(visibleTimeStart).toISOString(), endTime: moment(visibleTimeEnd).toISOString() })
+                ||
```

Comment:
That was my original approach, but then realized that it seems to matter to what specificity you are comparing the times. So, if I covered this case in the doTimeRangesOverlap, it would also mean that 1:00-2:00 would overlap with 2:00-3:00, which is not the case.